lib/Extension/ClassMover/Rpc/ReferencesHandler.php

```diff
@@ -268,19 +268,46 @@ private function performFindOrReplaceReferences(
     ) {
         switch ($symbolContext->symbol()->symbolType()) {
             case Symbol::CLASS_:
-                return $this->classReferences($filesystem, $symbolContext, $source, $replacement);
+                [$source, $references] = $this->classReferences($filesystem, $symbolContext, $source, $replacement);
+                break;
             case Symbol::METHOD:
-                return $this->memberReferences($filesystem, $symbolContext, ClassMemberQuery::TYPE_METHOD, $source, $replacement);
+                [$source, $references] = $this->memberReferences($filesystem, $symbolContext, ClassMemberQuery::TYPE_METHOD, $source, $replacement);
+                break;
             case Symbol::PROPERTY:
-                return $this->memberReferences($filesystem, $symbolContext, ClassMemberQuery::TYPE_PROPERTY, $source, $replacement);
+                [$source, $references] = $this->memberReferences($filesystem, $symbolContext, ClassMemberQuery::TYPE_PROPERTY, $source, $replacement);
+                break;
             case Symbol::CONSTANT:
-                return $this->memberReferences($filesystem, $symbolContext, ClassMemberQuery::TYPE_CONSTANT, $source, $replacement);
+                [$source, $references] = $this->memberReferences($filesystem, $symbolContext, ClassMemberQuery::TYPE_CONSTANT, $source, $replacement);
+                break;
+            default:
+                throw new \RuntimeException(sprintf(
+                    'Cannot find references for symbol type "%s"',
+                    $symbolContext->symbol()->symbolType()
+                ));
         }
 
-        throw new \RuntimeException(sprintf(
-            'Cannot find references for symbol type "%s"',
-            $symbolContext->symbol()->symbolType()
-        ));
+        return [$source, $this->sortReferences($references)];
+    }
+
+    public function sortReferences(array $fileReferences): array
+    {
+        // Sort the references for each file
+        \array_walk($fileReferences, function (array &$fileReference) {
```

Comment:
no need to use an FQN here?

Reply:
All the references reference the same name so I'm not sure on which FQN we could sort.
The only things we are provided with are the file path and the positions.
So we sort the references in each files by position and then all files by alphabetical order.

Reply:
i mean the `array_walk` doesn't need to be absolutely referenced?

Reply:
also should be private?

Reply:
ho yes, bad habit from work with the `\` and public well... It seems I'm out of my game :)